tutorials/10_DebuggingREMIND.md

```diff
@@ -174,6 +181,8 @@ Only the first five columns are of particular interest:
 - `RGmax`: if during phase 1-2 the sum of infeasibilities, it should always converge to a very small value (< 10e-7). If during phase 3/4 the reduced gradient (when small you are close to optimality).
 - `NSB`: the number of superbasic variables (basically the current number of degrees of freedom).
 
+Displaying the CONOPT output multiple times will show you which regions are still being solved and if they are making progress. If `Iter` does not change over time or has very high values (> 10 000), this can be a hint that CONOPT froze and the run should be restarted from the latest `fulldata.gdx`.
```

Comment:
>  If `Iter` does not change over time or has very high values (> 10 000), this can be a hint that CONOPT froze and the run should be restarted from the latest `fulldata.gdx`.

I would write
> If `Iter` does not change between consecutive calls to `conoptspy`, then CONOPT crashed.  If CONOPT is at unusually high iterations (> 10000), and there is little or no progress in the objective value, and the reduced gradient does not change or oscillates between values without making progress, then CONOPT is stuck somewhere in the solution space and won't get out.
> In both cases, abort the run and restart from the `fulldata.gdx` (the last feasible solution).